canonical-web-and-design/cookie-policy
# Add auto disappear functionality

## Done
Added self-destruct functionality to the notification.

## QA
- Run `gulp build`
- Open the index.html file
- Check the cookie policy self-destructs after 3 seconds
- Update the duration on in the index.html and refresh
- Check the duration matches your change
- Remove the duration and check it never goes away

## Details
Fixes https://github.com/canonical-webteam/cookie-policy/issues/3
